otcextensions/tests/functional/sdk/dns/__init__.py

```diff
@@ -0,0 +1,147 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import os
+import uuid
+
+import fixtures
+
+from otcextensions.tests.functional import base
+
+
+class TestDns(base.BaseFunctionalTest):
+    uuid_v4 = uuid.uuid4().hex[:8]
+    router_name = 'sdk-dns-test-router-' + uuid_v4
+    net_name = 'sdk-dns-test-net-' + uuid_v4
+    subnet_name = 'sdk-dns-test-subnet-' + uuid_v4
+    router_id = None
+    net_id = None
+    subnet_id = None
+
+    def setUp(self):
+        super(TestDns, self).setUp()
+        test_timeout = 3 * int(os.environ.get('OS_TEST_TIMEOUT'))
+        try:
+            self.useFixture(
+                fixtures.EnvironmentVariable(
+                    'OS_TEST_TIMEOUT', str(test_timeout)))
+        except ValueError:
+            pass
+        self.client = self.conn.dns
+
+    def create_network(self):
+        self.cidr = '192.168.0.0/16'
+        self.ipv4 = 4
+        self.dns_nameservers = ['100.125.4.25', '8.8.8.8']
+
+        network = self.conn.network.create_network(name=self.net_name)
+        self.assertEqual(self.net_name, network.name)
```

Comment:
it is not a thing which need to be tested here, is it neccessary?

Reply:
this is for ensure that resource successfully created